spec/models/foodstuff_spec.rb

```diff
@@ -1,5 +1,59 @@
 require 'rails_helper'
 
 RSpec.describe Foodstuff, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe "quantityカラムのバリデーション" do
+    context "OK" do
+      it "数字のみ(半角)であること" do
+      foodstuff = build(:foodstuff, :only_half_size_number)
+      foodstuff.valid?
+      expect(foodstuff).to be_valid
+      end
+
+      it "数字のみ(全角)であること" do
+        foodstuff = build(:foodstuff, :only_full_width_number)
+        foodstuff.valid?
+        expect(foodstuff).to be_valid
```

Comment:
`foodstuff.valid?` の結果はbooleanで帰ってきます。よって下記でもいいです。

```suggestion
        expect(foodstuff.valid?).to be true
```